react/components/DescriptiveCarousel/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import Carousel from '../Carousel/index';
+import Type from '../../styles/Type';
```

Comment:
Use absolute paths in JS. It might seem a bit verbose at first but it's much simpler to reason about and copy paste.

`import Type from 'react/styles/Type;`